persistence/mongodb/src/main/scala/net/liftweb/mongodb/Mongo.scala

```diff
@@ -126,6 +127,7 @@ object MongoDB {
     * and the use of getLastError.
     * See: http://docs.mongodb.org/ecosystem/drivers/java-concurrency/
     */
+  @deprecated("No longer relevant. See mongo-java-drivers's JavaDocs for details", "3.0")
```

Comment:
When we say it's not “relevant”, do we mean you don't need to wrap your code in this? If so, perhaps we can say “No longer needed” instead?